horace_core/sqw/file_io/sqw_formats_factory.m

```diff
@@ -45,10 +45,10 @@
         %
         % Rules to load/save different classes:
         % sqw2 corresponds to sqw file in indirect mode with varying efixed
-        written_types_ = {'sqw_old','sqw2','dnd','d0d_old','d1d_old','d2d_old','d3d_old','d4d_old'};
+        written_types_ = {'sqw', 'sqw_old','sqw2','dnd','d0d_old','d1d_old','d2d_old','d3d_old','d4d_old'};%,'sqw'};
```

Comment:
This has support for `sqw` and `sqw_old`, but the `if` on line 105 below only supports `sqw`.
Ia `sqw_old` support required at all?

Reply:
I think you mean line 195. My approach was that if an if was testing that the object was sqw_old, we should change it to sqw as that is what we want to treat going forward - that is what line 195 does. Here the aim was to support sqw_old when it occurs correctly so nothing breaks for no good reason. All the changes are the result of fixing a broken test. So sqw_old is retained in this list. When we remove sqw_old this can go, but I'm not breaking any tests at the moment to initiate a clearout.

Reply:
I wanted to confirm that the rest of the file and file handling routines were able to correctly handle both sqw and sqw_old if both options were included here; i.e are there other places where we have replaced sqw_old with sqw rather than sqw||sqw_old

Thinking here about changes in other files e.g. dnd_binfile_common as well as elsewhere in this one